Rocket.Chat/Views/Avatar/AvatarView.swift

```diff
@@ -16,7 +16,7 @@ final class AvatarView: UIView {
     var imageURL: URL? {
         didSet {
             if let imageURL = imageURL {
-                let options: SDWebImageOptions = [.retryFailed, .scaleDownLargeImages, .highPriority, .refreshCached]
+                let options: SDWebImageOptions = [.retryFailed, .scaleDownLargeImages, .highPriority, .cacheMemoryOnly]
```

Comment:
Is this fixing a bug? 🤔

Reply:
Kind of. This is the regression I mentioned... it prevents the app from requesting the avatar over HTTP on every cell and hitting the cache. We need a better way to deal with the request before we rely on HTTP cache.